packages/example/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@celo-tools/use-contractkit": "1.1.0",
+    "@celo-tools/use-contractkit": "1.2.0",
```

Comment:
As I did my PR, would be nice to keep this package's version in sync with the use-ck version it's using